nx.json

```diff
@@ -115,5 +117,6 @@
       }
     },
     "releaseTagPattern": "v{version}"
-  }
+  },
+  "useInferencePlugins": false
```

Comment:
automatically added by Nx during migrations. probably to avoid surprises of Project Crystal.

Reply:
Probably for the best :grin: